lib/rucio/rse/protocols/protocol.py

```diff
@@ -120,7 +122,10 @@ def _get_path(self, scope, name):
         hstr = hashlib.md5('%s:%s' % (scope, name)).hexdigest()
         if scope.startswith('user') or scope.startswith('group'):
             scope = scope.replace('.', '/')
-        return '%s/%s/%s/%s' % (scope, hstr[0:2], hstr[2:4], name)
+        if self.attributes.get('determinism_type', 'default') == 'identity':
```

Comment:
For this part I really wonder if it would be beneficial if it comes from the policy and not the determinism_type from the RSE. e.g. for CMS the 'default' policy could be defines as `'%s/%s' % (scope, name)` while for ATLAS it is the hash part. Both have advantages, but I am mostly wondering if you forget to set the determinism_type for a newly created RSE you will get the hashed pathes until you fix it, while by setting the default via the policy that won't happen. Any preferences?